parts/linux/cloud-init/artifacts/components.json

```diff
@@ -1233,15 +1233,20 @@
         "default": {
           "current": {
             "versionsV2": [
+              {
+                "k8sVersion": "1.31",
+                "renovateTag": "OCI_registry=https://mcr.microsoft.com, name=oss/binaries/kubernetes/azure-acr-credential-provider",
+                "latestVersion": "v1.31.2"
+              },
               {
                 "k8sVersion": "1.30",
                 "renovateTag": "OCI_registry=https://mcr.microsoft.com, name=oss/binaries/kubernetes/azure-acr-credential-provider",
-                "latestVersion": "v1.30.7"
+                "latestVersion": "v1.30.8"
               },
               {
                 "k8sVersion": "1.29",
-                "renovateTag": "OCI_registry=https://mcr.microsoft.com, name=oss/binaries/kubernetes/azure-acr-credential-provider",
-                "latestVersion": "v1.29.11"
+                "renovateTag": "OCI_regitry=https://mcr.microsoft.com, name=oss/binaries/kubernetes/azure-acr-credential-provider",
+                "latestVersion": "v1.29.12"
```

Comment:
@mainred are windows binaries included in those MCR binaries?

Reply:
Yes. both windows and linux credential provider binaries are in MCR